letsdrill.rb

```diff
@@ -1,24 +1,28 @@
-def get_letter_grade(integer)
- 
-  #Put your code here!
-
+def get_letter_grade(score)
+  case score
+  when 90..100
+    return "A"
+  when 80..90
+    return "B"
+  when 70..80
+    return "C"
+  when 60..70
+    return "D"
+  end
+  return "F"
 end
 
-def shortest_string(array)
-
-  #Put your code here!
-
+def shortest_string(strings)
+  strings.reduce { |m, s| s.length < m.length ? s : m }
```

Comment:
I think the min_by method would be more clear and concise here. Thanks for the recommendation!